src/SourceMapToolkit.CallstackDeminifier/StackTraceDeminifier.cs

```diff
@@ -21,7 +21,7 @@ internal StackTraceDeminifier(IStackFrameDeminifier stackFrameDeminifier, IStack
 		/// <summary>
 		/// Parses and deminifies a string containing a minified stack trace.
 		/// </summary>
-		public DeminifyStackTraceResult DeminifyStackTrace(string stackTraceString, bool preferSourceMapsSymbols = false)
+		public DeminifyStackTraceResult DeminifyStackTrace(string stackTraceString, bool preferSourceMapsSymbols = false, bool fixOffByOne = false)
```

Comment:
Could we name this something a little clearer? Maybe something like `useCallerName`?

Given this is also dependent on `preferSourceMapsSymbols`'s state, it may be good to add an explanation in the comment above.

Reply:
Updated